### ISSUE TITLE
Visual problem with bootstrap 3

### ISSUE THREAD

**Issue by woodyalan** [nathanreyes/v-calendar#329]
Hello!

Visually it does not look like the examples. Any suggestions on what can be?

**main.js**
```
import Vue from 'vue';
import VCalendar from 'v-calendar';

Vue.use(VCalendar);
```

**InputDatePicker.vue**
```
<template>
  <v-date-picker
      mode="single"
      :first-day-of-week="1"
      is-dark
    ></v-date-picker>
</template>

<script>
export default {
  name: "form-group-date-picker"
};
</script>
```

![Provas Grupo Integrado](https://user-images.githubusercontent.com/1576312/58730182-c7787d80-83c1-11e9-8202-2284eec02e85.png)

**Comment by adamorlowskipoland** [comment]
Have the same problem, also with bootstrap 3 in the project.

**Comment by np9999** [comment]
I've had the same issue. Bootstrap 3 adds "font-size:10px" CSS to the <html> tag on the page, and v-calendar is inheriting that style. To fix this, I added my own css on the header of the page where I implemented the calendar.

`<style>
        html {
            font-size: 14px;
        }
    </style>`

**Comment by nathanreyes** [comment]
Yeah I think I'll just convert all rem dimensions to px to avoid this inconsistencies across css frameworks.

**Comment by nathanreyes** [comment]
Fixed as of `v1.0.0-beta.15`
